cost/turbonomics/scale_virtual_machines_recommendations/aws/turbonomics_scale_virtual_machines.pt

```diff
@@ -0,0 +1,396 @@
+name "Turbonomics Scale Virtual Machine Recommendations AWS"
+rs_pt_ver 20180301
+type "policy"
+short_description "POC turbonomics policy for scaling virtual machines [README](https://github.com/flexera-public/policy_templates/tree/master/cost/turbonomics/scale_virtual_machines_recommendations) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "1.0",
+  provider: "AWS",
+  source: "Turbonomics",
+  service: "Compute",
+  policy_set: "Scale VMs",
+  recommendation_type: "Usage Reduction"
+)
+
+##################
+# Parameters   #
+##################
+
+## once functionality allows, this will contain "ALL", "AWS", "Azure Subscription", "GCP Project" and we will remove the policies from thier folders
+parameter "param_provider" do
+  type "string"
+  label "Recommendation Vendor"
+  description "authorization cookie pulled from manual source. Allows \"\", GCP Project, AWS, or Azure Subscription as a value"
+  allowed_values "AWS"
+end
+
+parameter "auth_cookie" do
+  type "string"
+  label "Authorization Cookie"
+  no_echo true
+  description "authorization cookie pulled from manual source"
```

Comment:
Change this to "Valid authorization cookie".

Reply:
Done!